src/browser-polyfill.js

```diff
@@ -336,5 +336,7 @@ if (typeof browser === "undefined") {
     return wrapObject(chrome, staticWrappers, apiMetadata);
   };
 
-  this.browser = wrapAPIs();
+  module.exports = wrapAPIs();
```

Comment:
Ooh, that's a good point about "commonjs" mode causing eslint to be too lenient. I've pushed up https://github.com/mozilla/webextension-polyfill/pull/17/commits/35e8e7a465256945367c16216508593e6024837e with the suggested changes. Of course, if we go back to renaming the global, this will have to be reverted/etc.